### PR TITLE
Partial (1/2) spanish translation ch10-01-syntax.md

### PR DESCRIPTION
**Pull request by jossue33**
Number of words: 1073
Line: 1-259

### CODE REVIEW

File: second-edition/src/ch10-01-syntax.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-methods using generics, and at the end of this section we’ll discuss the
-performance of code using generics.
+Usando los genéricos donde usualmente ponemos tipos, como en firmas de función
+o estructuras, nos deja crear definiciones que podemos usar para tipos 
+de datos concretos. Miremos cómo definir funciones, estructuras, enum y
+métodos usando genéricos, y al final de esta sección discutiremos el
+desempeño del código usando genéricos.
 
-### Using Generic Data Types in Function Definitions
+### Usando tipos de datos genéricos en la definición de funciones
```
Comment (thecodix) [inline second-edition/src/ch10-01-syntax.md]: Please translate titles with Camel Case, like in the english version :)

**Comment by jossue33** [comment]
@thecodix espero así sea.

**Comment by thecodix** [comment]
Please answer comments in english

**Comment by thecodix** [comment]
machine translated

**Pull request merged**